### DISCUSSION TITLE
Is there a way to dump the traffics/msgs to a file on disk?

### DISCUSSION THREAD

**Discussion by t0ny-peng** [eclipse-iceoryx/iceoryx#1993] [General]
Hi guys. I'm writing an application that uses Iceoryx for communication. Just wonder if there's a recommended way to dump all the traffics/messages to a file for further debugging? Something similar to `ros2 bag record`. 
Thanks!

**Comment by t0ny-peng** [comment]
Even saving the raw data (memory dump of that message) to the disk would be helpful.

**Reply by elfenpiff** [reply]
We did not have implemented such a feature but we have an rmw binding for ros2. When you use this you should be able to use `ros2 bag record` directly, see https://github.com/ros2/rmw_iceoryx
